storage/remote/write_handler.go

```diff
@@ -620,8 +620,8 @@ func (rw *rwExporter) Capabilities() consumer.Capabilities {
 type otlpWriteHandler struct {
 	logger *slog.Logger
 
-	defaultConsumer consumer.Metrics // only cumulative
-	d2cConsumer     consumer.Metrics // delta capable
+	defaultConsumer consumer.Metrics // accepts both cumulative and delta metrics, stores deltas as-is
+	d2cConsumer     consumer.Metrics // only accepts delta metrics, converts them to cumulative
```

Comment:
d2c accepts both delta and cumulative actually, but transforms deltas into cumulative